README.md

```diff
@@ -255,27 +283,32 @@ class KblSampleTest {
         final KblSpecialWireOccurrence specialWireOccurrence = null; // determine kbl wire occurrence
 
         assertThat(specialWireOccurrence)
-            .hasSpecialWireId("1111")
-            .hasPart(wire)
-            .satisfies(w ->
-                assertThat(w.getLengthInformations()).hasSize(1)
-                    .satisfies(
-                        // Consumer
-                        l ->
-                            assertThat(l)
-                                .hasLengthType("Production")
-                                .satisfies(v ->
-                                    assertThat(v.getLengthValue())
-                                        .hasUnitComponent(unitMM)
-                                        .hasValueComponent(0.0d)),
-                        // Index
-                        atIndex(0))
-            );
+                .hasSpecialWireId("1111")
+                .hasPart(wire)
+                .satisfies(w ->
+                        assertThat(w.getLengthInformations()).hasSize(1)
+                                .satisfies(
+                                        // Consumer
+                                        l ->
+                                                assertThat(l)
+                                                        .hasLengthType("Production")
+                                                        .satisfies(v ->
+                                                                assertThat(v.getLengthValue())
+                                                                        .hasUnitComponent(unitMM)
+                                                                        .hasValueComponent(0.0d)),
+                                        // Index
+                                        atIndex(0))
+                );
     }

```

Comment:
Perhaps those additional indentations should be removed for a better visibility.

Reply:
The assertion is removed in the next branch